images/devtools-kubernetes-v1beta1/context/Dockerfile

```diff
@@ -24,7 +24,12 @@ COPY ./magefiles /magefiles
 WORKDIR /magefiles
 RUN \
     mage -compile  ./mage-k8s
-
+    
+WORKDIR /build
+RUN \
+    curl -LO https://github.com/kyverno/kyverno/releases/download/v1.12.0/kyverno-cli_v1.12.0_linux_x86_64.tar.gz && \
```

Comment:
Seems like kyverno-cli is also available in docker image, so it could be added the same way other binaries are added to the image: https://github.com/kyverno/kyverno/pkgs/container/kyverno-cli